tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-pascasarjana.csl

```diff
@@ -0,0 +1,623 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="id-ID">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Tesis dan Disertasi Universitas Negeri Yogyakarta Pendidikan Pascasarjana</title>
+    <title-short>TD-UNY-PPS</title-short>
+    <id>http://www.zotero.org/styles/tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-pascasarjana</id>
+    <link href="http://www.zotero.org/styles/tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-pascasarjana" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
+    <link href="https://drive.google.com/open?id=0B1fB-hcWFL4_SHNETkdrSmgwSGM" rel="documentation"/>
```

Comment:
documentation has been added.

Reply:
[Pedoman Tesis dan Disertasi 2017_lengkap.pdf](https://github.com/citation-style-language/styles/files/1385096/Pedoman.Tesis.dan.Disertasi.2017_lengkap.pdf)
@rmzelle do yo mean like this?